dashboard: update console media to v6.5 (#12083)
pingcap/docs: dashboard/dashboard-cluster-info.md

Click **Hosts** to view the list of hosts:

![Host list](/media/dashboard/dashboard-cluster-info-hosts-v650.png)

This host list shows the running status of hosts that correspond to all instances of TiDB, TiKV, PD, and TiFlash components in the cluster.

The list includes the following information:

- Host Address: The Host IP address.
- CPU: The number of logical cores of the host CPU.
- CPU Usage: The user-mode and kernel-mode CPU usage in the current 1 second.
- Memory: The total physical memory size of the host.
- Memory Usage: The current memory usage of the host.

> **Note:**
>
> The host list information is provided by each instance process, so when all instances on the host are down, the host information is not displayed.

## Disk list

Click **Disks** to view the list of disks:

![Disk list](/media/dashboard/dashboard-cluster-info-disks-v650.png)

> This disk list shows the running status of disks that correspond to all instances of TiDB, TiKV, PD, and TiFlash components in the cluster.

Reviewer: ```suggestion
This disk list shows the status of disks on which the TiDB, TiKV, PD, and TiFlash instances run.
```
